spec/rails_helper.rb

```diff
@@ -35,6 +37,8 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.include(Shoulda::Matchers::ActiveModel, type: :model)
```

Comment:
@isabloCh You don't need these 2 lines. Documentation says:

**If you are using ActiveModel or ActiveRecord outside of Rails** and you want to use model matchers in certain example groups, you'll need to manually include them.